tests/layer_tests/pytorch_tests/test_dot.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+
+from pytorch_layer_test_class import PytorchLayerTest
+
+class TestDot(PytorchLayerTest):
+    def _prepare_input(self, inputs, dtype=None):
+        import numpy as np
+        return [np.array(i).astype(dtype) for i in inputs]
+
+    def create_model(self, dtype=None):
+        import torch
+        dtype_map = {
+            "float32": torch.float32,
+            "float64": torch.float64,
+            "int64": torch.int64,
+            "int32": torch.int32,
+            "uint8": torch.uint8,
+            "int8": torch.int8,
+        }
+
+        class aten_dot(torch.nn.Module):
+            def __init__(self, dtype) -> None:
+                super().__init__()
+                self.dtype = dtype
+
+            def forward(self, tensor1, tensor2):
+                return torch.dot(tensor1, tensor2)
```

Comment:
I don't know if you ran tests locally, but I would expect this test to fail due to unknown input types. I would recommend using `to`, e.g. `tensor1.to(torch.float32)` to solve issue with unknown types.

Reply:
> I don't know if you ran tests locally, but I would expect this test to fail due to unknown input types. I would recommend using `to`, e.g. `tensor1.to(torch.float32)` to solve issue with unknown types.

I'm having problem on running tests locally (I've talked with you about this also in the implementation of expm1 op). I'm trying to solve this issue.